bin/console.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:running_on_dart/src/api/jwt_middleware.dart';
+
+enum Commands {
+  generateAdminJwt('generate-admin-jwt');
+
+  final String name;
+  const Commands(this.name);
+
+  static Commands byName(String name) {
```

Comment:
FWIW, [`Enum.byName`](https://api.dart.dev/stable/3.5.4/dart-core/EnumByName/byName.html) already exists in an extension